src/promptflow-evals/promptflow/evals/evaluators/fluency/__init__.py

```diff
@@ -4,20 +4,25 @@
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
 
+from typing import Optional
+
 from promptflow.client import load_flow
 from promptflow.entities import AzureOpenAIConnection
 from pathlib import Path
 
 
 class FluencyEvaluator:
-    def __init__(self, model_config: AzureOpenAIConnection, deployment_name: str):
+    def __init__(self, model_config: AzureOpenAIConnection, deployment_name: str,
+                 log_level: Optional[int] = None):

```

Comment:
Is this `log_level` requirement only for evalualtors? Have we considered overriding the log level inside evalualtors?

Reply:
According to the task: https://msdata.visualstudio.com/Vienna/_workitems/edit/3096911/ we want to avoid evaluator to print the Promptflow logs, or at least make it configurable.